{"name": "test_login_with_valid_email_and_invalid_password_Duplicate", "status": "passed", "start": 1695709747471, "stop": 1695709750432, "uuid": "752b7223-3962-41bf-b9b1-1a7ec3ca58af", "historyId": "9581a2f8ea7f6829aae721e266b65c65", "testCaseId": "9581a2f8ea7f6829aae721e266b65c65", "fullName": "Tests.test_Login(duplicate).TestPanarayLoginDuplicate#test_login_with_valid_email_and_invalid_password_Duplicate", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login(duplicate)"}, {"name": "subSuite", "value": "TestPanarayLoginDuplicate"}, {"name": "host", "value": "WONIT11L32CV6Z"}, {"name": "thread", "value": "12312-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login(duplicate)"}]}